app/src/sharedTest/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentTest.kt

```diff
@@ -158,7 +158,7 @@ class ConceptCardFragmentTest {
   }
 
   @Test
-  fun testConceptCardFragment_openDialogFragment0_checkSkillAndExplanationAreDisplayedWithoutRichText() { // ktlint-disable max-line-length
+  fun testCCFragment_openDialogFragment0_checkSkillAndExplanationDisplayedWithoutRichText() {
```

Comment:
Alternate suggestion which works just by removing `Fragment`

`fun testConceptCard_openDialog0_checkSkillAndExplanationAreDisplayedWithoutRichText() {`

Reply:
Same things applies for this entire file?

Reply:
@varunvjha Always reply to these comments.